src/main/java/frc/robot/Constants.java

```diff
@@ -148,6 +152,7 @@ public static class kManipulator {
     public static final double WRIST_STOP_SPEED = 1.0;
 
     public static final int ERROR_LIMIT = 1;
+    public static final int MAX_ERROR = 5;
```

Comment:
154 and 155 have similar names, its confusing what the difference between them, spesificy in comments that one of them is for the absolute encoder. rename 155 to PID_ERROR